packages/terra-form-textarea/docs/MigrationGuide.md

```diff
@@ -0,0 +1,19 @@
+# Migrating from terra-form's textarea Component to terra-form-textarea
+
+The following sections describe how to migrate from the textarea component provided in the terra-form package to the textarea component provided in the terra-form-textarea package. Terra-form-textarea provides enhanced API and layout to begin the first phase of the OSC uplift.
```

Comment:
*OCS

I'm wondering if we should really include this in the documentation though, any consumers outside of cerner would have no idea what this means. Thoughts?

Reply:
I agree as well. I have removed the migration guide, so it's no longer here. https://github.com/cerner/terra-core/pull/982/commits/802777623b56e1099bdf699936a4de4e2853d6ff

In addition, I'll give the textarea a new text description. I think when we talk about OCS, we should just say the component is themeable. How does that sound?